private-key-compromise/src/utils.ts

```diff
@@ -27,6 +33,15 @@ export type AlertedAddress = {
   timestamp: number;
 };
 
+// Left two properties `optional` for
+// backwards compatibility
+export type UpdatedAlertedAddress = {

```

Comment:
did you leave the old `AlertedAddress` as it is on purpose? because if we're going to use this one, we can just update the old `AlertedAddress` with the new properties

Reply:
My end intention wasn't to have to `type`s, but I left both for now for the beta deployment. In the end, it was meant to have just one. Will make a note to not leave both. Thanks for pointing that out. 👍